### ISSUE TITLE
Add configuration UI

### ISSUE THREAD

**Issue by joboet** [supercoder186/XSharedCockpit#3]
At the moment, the IPs are hardcoded, and the mode is set via toggle commands.
**Fix:** Add a configuration UI with ImGUI or another GUI library.

**Comment by supercoder186** [comment]
Yes... I definitely need to implement a GUI for this plugin. Currently, I use the config file to specify the IP address and ports. I will also update the README for better documentation :)

**Comment by supercoder186** [comment]
Would you be interested in aiding me with the addition of imgui to this plugin? I have been experiencing issues with getting it to build properly for the windows platform with the templates provided by the X-Plane community (ImgWindow and imgui4xp)
